src/App.tsx

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
 import './App.scss';
+import { GoodsList } from './GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+type State = {
+  goods: Good[];
+};
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
+
+  handleChange = (callback: () => Promise<Good[]>) => {
```

Comment:
```suggestion
  loadGoods = (callback: () => Promise<Good[]>) => {
```
назва handleChange мала б нам казати, що якись інпут має змінитись. Не дуже доречно в даному випадку